datamonster_api/lib/company.py

```diff
@@ -13,14 +13,27 @@ def __init__(self, id_, ticker, name, uri, dm):
         self.uri = uri
         self.dm = dm
 
+    def __repr__(self):
+        ticker_str = '[{}] '.format(self.ticker) if self.ticker else ''
+        return '<{}: {}{}>'.format(self.__class__.__name__, ticker_str, self.name)
```

Comment:
nice!!!

Reply:
thanks!
Motivation: a 'ticker' item is added to 'split_combination' subdicts, based on the 'section_pk' items. Tickers are friendlier/more meaningful to users than pk's, but not every "company" has one. Since the corresponding companies are already loaded, I want to change the "ticker" item to "company" (or list of) -- Company objects corresponding to the pk's in 'section_pk'. 
With this repr, tickers wouldn't be missed.